pkgs/top-level/all-packages.nix

```
@@ -20289,7 +20289,8 @@ with pkgs;
   texinfo6_5 = callPackage ../development/tools/misc/texinfo/6.5.nix { }; # needed for allegro
   texinfo6_7 = callPackage ../development/tools/misc/texinfo/6.7.nix { }; # needed for gpm, iksemel and fwknop
   texinfo6 = callPackage ../development/tools/misc/texinfo/6.8.nix { };
-  texinfo7 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
+  texinfo7_0 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
+  texinfo7 = callPackage ../development/tools/misc/texinfo/7.1.nix { };
```

Let's be more explicit:

```suggestion
  texinfo7_0 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
  texinfo7_1 = callPackage ../development/tools/misc/texinfo/7.1.nix { };
  texinfo = texinfo7_1;
```

---

@AndersonTorres would like to commit your suggestion, yet the message I receive over and over again when trying to do so is: `This diff has recently been updated. Refresh and try again.`

Are you okay with me making the change locally and including you suggestion in a single commit?

---

OK to me.

---

> Are you okay with me making the change locally and including you suggestion in a single commit?

It should have been squashed anyway